### ISSUE TITLE
Anyone interested in joining forces and fixing the problems together?

### ISSUE THREAD

**Issue by vogtmh** [the0neyouseek/MonitorControl#47]
Hi there,

I found this wonderful tool two weeks ago and it solved the one problem for me that Apple couldn't address: controlling my screens volume over HDMI.

But it has some flaws that are already documented in the issues and it doesn't seem to get updated for some months now. What do you think of fixing those bugs and if everything's alright adding the app to the Appstore for free, so everyone can profit from a simple app that solves a major macOS issue? ;-) 

To be honest, I'm quite new to Xcode development. I can compile and build, I can rewrite stuff, but I can't write an app on my own. I can to do testing and graphical design as well. What I already did was taking an old version of this app (without the key controls) and reducing it to a simple volume slider for single screen only. This works absolutely stable for me and wastes no CPU time, so I can just keep it in my startup.

So what do you think? Anyone interested? :-) 

- Max

![image](https://user-images.githubusercontent.com/37808063/44285085-3e45ef00-a264-11e8-893b-c334583298c1.png)

**Comment by carlosfilhoadm** [comment]
This looks really slick and functional.

Is there a release of this version somewhere for downloading?

**Comment by vogtmh** [comment]
I've forked the original version and added my "hacked" files to it: https://github.com/vogtmh/MonitorControl.OSX

**Comment by kocinko** [comment]
Thank you very much. I have resolved to make the Mojave Volume setting permanent.

**Comment by vogtmh** [comment]
I have used a former version of MonitorControl which didn't include any key controls. The repo from the0neyouseek includes no third party dependencies and I didn't want to mess around too much with it. Maybe I'll do a new version in the next weeks as I really miss the controls, too ;-)

**Comment by vogtmh** [comment]
So I downloaded the repo and did all the steps described in the Readme. First, I had to remove the code signing identities, now it cannot use MASPreferences.. might take a while to figure out the problem. That's why avoided this repo in the first place :-/

**Comment by vogtmh** [comment]
Find my new simplified version here: https://github.com/mavodev-de/ScreenVolume2

I even managed to get CPU usage down to nearly 0 % by turning of the continuous polling.

**Comment by vogtmh** [comment]
I might do that, but to be honest: I'm not a real developer (yet!) :-)

Removing / changing code is something completely different than write some new on your own. I just adapted the tool to my needs and fixes some really annoying problems. 

All the minor stuff will be done one by one once I find the time and passion. However, I just updated the app to run fine on Mojave, so you might have a look ;-) Next thing will be the initial state of the slider (currently 0, updated to the correct value once you hit a media key). 

If it takes too long for you, feel free to fork mine and start coding yourself - just like I did :-D
